hazelcast/src/main/resources/hazelcast-config-5.3.json

```diff
@@ -4989,6 +4999,34 @@
           }
         }
       }
+    },
+    "AltoConfig": {
```

Comment:
Similar to my comment in the Spring XSD, I believe we should add some documentation to those

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/034070382c199cea19e7f577912ba529eaf055f4